haret/src/vr/states/common/normal.rs

```diff
@@ -67,12 +67,11 @@ pub fn handle_start_view<T: State>(state: T,
 }
 
 pub fn handle_get_state<T: State>(state: T,
-                                  msg: GetState,
+                                  GetState { epoch, view, op }: GetState,
```

Comment:
We've had matching in fn args for years! I just checked, and we have one test for this from 2013 :)